Justfile

```diff
@@ -13,8 +13,8 @@ check-ruff:
     ruff check probe_src
 
 check-mypy:
-    MYPYPATH=probe_src mypy --strict --package arena
-    MYPYPATH=probe_src mypy --strict --package probe_py
+    mypy --strict --package probe_py.manual
+    mypy --strict --package probe_py.generated
```

Comment:
The `--package` flag seems to only work if the `pypackage.toml` file is in `$PWD`.
(`just` is smart enough to change the `$PWD` if you use it "at a distance" with `-f`)
```suggestion
    env -C ./probe_src/python mypy --strict --package probe_py.manual
    env -C ./probe_src/probe_frontend/python mypy --strict --package probe_py.generated
```

Reply:
Setting `$MYPYPATH` in the setup script is imho cleaner (it's set to almost the same thing we set `$PYTHONPATH` to.